OpenRA.Mods.RA/Traits/LeavesHusk.cs

```diff
@@ -19,8 +20,24 @@ namespace OpenRA.Mods.RA.Traits
 	public class LeavesHuskInfo : ITraitInfo
 	{
 		[ActorReference, FieldLoader.Require]
+		[Desc("Actor to spawn on death.")]
 		public readonly string HuskActor = null;
 
+		[Desc("Probability the husk actor spawns.")]
+		public readonly int Probability = 100;
+
+		[Desc("Owner of the husk actor. Leave empty to set" +
+			"the husk actor's owner to the dead actor's owner." +
+			"Use 'Killer' to set the husk actor's owner to the killer's owner.")]
+		public readonly string Owner = null;
```

Comment:
Relying on magic values is a bit ugly.  How about adding an `OwnerType` enum with values `Spawner`, `Killer`, or `InternalName`, and then having a string `OwnerInternalName` property for the latter option.